### ISSUE TITLE
NullConverter not a class in rspec 3.3.0-3.3.2

### ISSUE THREAD

**Issue by chhsiung** [rspec/rspec-legacy_formatters#16]
Getting this error when running with rspec 3.3.0 and rspec-legacy_formatters 1.0.0:
```
          TypeError:
            NullConverter is not a class
          # ./vendor/bundle/ruby/2.2.0/gems/rspec-legacy_formatters-1.0.0/lib/rspec/legacy_formatters/snippet_extractor.rb:8:in `<class:SnippetExtractor>'
```

In rspec-core 3.3.0-3.3.2, the NullConverter class has changed to a module.

**Comment by chhsiung** [comment]
thanks for the fix @JonRowe! any idea on when you guys will be cutting a new version of the gem?

**Comment by myronmarston** [comment]
@JonRowe I just gave you push access for this gem (along with the other rspec gems) to rubygems.org so you should be able to cut this now :).

**Comment by JonRowe** [comment]
I can't sign it @myronmarston

**Comment by myronmarston** [comment]
Released as 1.0.1.
